recipe/deploy/lock.php

```diff
@@ -29,3 +29,8 @@
 task('deploy:unlock', function () {
     run("rm -f {{deploy_path}}/.dep/deploy.lock");//always success
 });
+
+desc('Checks whether deploy is locked');
+task('deploy:islocked', function () {
```

Comment:
hmm tasks cannot return a value it seems. how to provide this api in a correct way?